tools/package

```diff
@@ -0,0 +1,236 @@
+    echo "Must have jq installed! Check your PATH"
+    exit 2
+fi
+
+if ! command -v fpm; then
+    echo "Must have fpm installed! Check your PATH"
+    exit 2
+fi
+
+
+if [ -z "${version}" ]; then
+    if ! version="$(cargo metadata --no-deps --format-version 1 | jq -r '.packages[0].version')"; then
+        echo "${version}"
+        echo "Failed to read extension version from Cargo metadata!"
+        exit 1
+    fi
+fi
+
+if [ -z "${pg_version}" ] && [ -z "${pg_config}" ]; then
+    echo "Must provide either --pg-version or --pg-config!"
+    exit 2
+fi
+
+mkdir -p "${out_dir}"
+if [ ! -d "${out_dir}" ]; then
+    echo "Invalid --out-dir '${out_dir}', directory doesn't exist!"
+    exit 2
+fi
+
+if [ -n "${pg_config}" ]; then
+    if config_version="$("${pg_config}" --version | cut -d' ' -f2 | cut -d '.' -f1)"; then
+        if [ -n "$pg_version" ]; then
+            if [[ ! "$config_version" = "$pg_version" ]]; then
+                echo "Mismatched --pg-config and --pg-version"
+                echo "  Expected version:   ${pg_version}"
+                echo "  Got from pg_config: ${config_version} "
+                exit 1
+            fi
+        else
+            pg_version="${config_version}"
+        fi
+    else
+        echo "$config_version"
+        echo "Unable to obtain version from pg_config!"
+        exit 1
+    fi
+fi
+
+# If no package format was set, select an appropriate one for the current OS
+if [ -z "$pkg_type" ]; then
+    case "$os_name" in
+        debian|ubuntu)
+            pkg_type="deb"
+            ;;
+        rhel|centos|fedora)
+            pkg_type="rpm"
+            ;;
+        *)
+            pkg_type="tar"
+            ;;
+    esac
+fi
+# Validate the selected package format
+case "$pkg_type" in
+    deb|rpm|tar) ;;
+    *)
+        echo "Invalid package type selected '${pkg_type}'. Must be one of deb|rpm|tar"
+        exit 2
+        ;;
+esac
+
+pg_package=
+case "$os_name" in
+    debian|ubuntu)
+        pg_package="postgresql-${pg_version}"
+        ;;
+
+    rhel|centos|fedora)
+        pg_package="postgresql${pg_version}-server"
+        ;;
+
+    *)
+        pg_package="postgresql${pg_version}-server"
+        ;;
+esac
+
+pg_config_dir=
+if [ -n "${pg_config}" ]; then
+    pg_config_dir="$(dirname "${pg_config}" && pwd -P)"
+else
+    case "$os_name" in
+        debian|ubuntu)
+            pg_config_dir="/usr/lib/postgresql/${pg_version}/bin"
+            ;;
+        rhel|centos|fedora)
+            pg_config_dir="/usr/pgsql-${pg_version}/bin"
+            ;;
+        *)
+            echo "Unable to detect pg_config, please set --pg_config"
+            exit 1
+            ;;
+    esac
+fi
+
+
+echo "Building.."
+cd "${ROOT_DIR}"
+
+PATH="${pg_config_dir}:${PATH}" cargo pgx package
+
+echo "Post-processing build.."
+cargo run --manifest-path tools/post-install/Cargo.toml -- --dir "target/release/promscale-pg${pg_version}"
+
+echo "Packaging.."
+fpm --log info -s dir -t "${pkg_type}" \
```

Comment:
should we add --meta-file flag and a debian directory with the copyright file like https://github.com/timescale/release-build-scripts/blob/master/debian/timescaledb-base-deb/debian/copyright

Reply:
Sure, can do!